### PR TITLE
Added Backup and Restore doc

### PR DESCRIPTION
**Pull request by krishnaraoch**

### CODE REVIEW

File: learn/wme-backup-restore/wme-backup-restore-introduction.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "WME-Backup-Restore"
+id: "wme-backup-restore-introduction"
+sidebar_label: "Introduction"
+---
+---
+## Introduction
+
+WaveMaker uses separate dedicated directory for storing data, which is easy to take backup and restore
```
Comment (krishnaraoch) [inline learn/wme-backup-restore/wme-backup-restore-introduction.md:9]: WaveMaker Platform Stores it's state into disk(specific directories as mentioned in the below images). WaveMaker Platform administrtors can take backups of those disk/directories and can restore to any previous state.

In this document we will learn how to take backups and restore in various infrastructure providers.

File: learn/wme-backup-restore/wme-backup-restore-introduction.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
```
Comment (krishnaraoch) [inline learn/wme-backup-restore/wme-backup-restore-introduction.md:10]: Platform Direcotory Structure Image
StudioWorkspace Direcotory Structure Image (Dont use External Instance. Use StudioWorkspace Instance or AppDeployment Instance).. these images should have all directories used by Platform, irresepective of state
Explain brifly about above  images. Explain about how do they can allocate disks/volumes for each directory.
Current image are not looking good. remove italic. little low color. work with swetha if needed

Comment (krishnaraoch) [reply learn/wme-backup-restore/wme-backup-restore-introduction.md:10]: What to be Backup
When to Restore
write only bullet points

File: website/sidebars.json [hunk 3]
```diff
@@ -556,10 +556,42 @@
...
-},
-
+  ],
+  
+  "Backup And Restore":
+      [
+        "wme-backup-restore/wme-backup-restore-introduction",
+	{
+		"type": "subcategory",
+      "label": "Platform",
```
Comment (krishnaraoch) [inline website/sidebars.json:566]: each section Backup & Restore on ESXi like that.
In a single file

File: learn/wme-backup-restore/backup/externalinstance/backup-aws.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Backup on AWS"
+id: ""
+sidebar_label: "Backup on AWS"
+---
+---
+
+
+Stop WME  EC2 external instance and detach network interface [follow the steps given here](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#detach_eni), make a note of interface ID or eni ID.
```
Comment (krishnaraoch) [inline learn/wme-backup-restore/backup/externalinstance/backup-aws.md:9]: write like steps. Bullet points

File: learn/wme-backup-restore/backup/externalinstance/backup-esxi.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Backup on ESXI"
+id: ""
+sidebar_label: "Backup on ESXI"
+---
+---
+
+
```
Comment (krishnaraoch) [inline learn/wme-backup-restore/backup/externalinstance/backup-esxi.md:8]: VMware ESXi ESXi : not ESXI
Give some intro always.
for ex: On VMware ESXi we can take backups in two differnt ways. 
1. Take backup of directory from inside the Platform Instance
    - What was the command and process
2. Take snapshots using VMware ESXi console.
  - Explain the process first 
- for more info go to this link

File: learn/wme-backup-restore/backup/externalinstance/backup-aws.md (hunk 4 above)
Comment (krishnaraoch) [inline learn/wme-backup-restore/backup/externalinstance/backup-aws.md:9]: In any Provider we can take backups from directories as well ?
so you can mention it and write the same like ESxi

File: learn/wme-backup-restore/restore/externalinstance/restore-esxi.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Restore on ESXI"
+id: ""
+sidebar_label: "Restore on ESXI"
+---
+---
+
+
+Launch the virtual machine with same ip address and copy the /data directory to new instance
```
Comment (krishnaraoch) [inline learn/wme-backup-restore/restore/externalinstance/restore-esxi.md:9]: This restore info will move to same file.

File: learn/wme-backup-restore/backup/platform/backup-aws.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Backup on AWS"
+id: ""
+sidebar_label: "Backup on AWS"
+---
+---
+
+
+Stop WME  EC2 instance and detach network interface [follow the steps given here](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#detach_eni), make a note of interface ID or eni ID.
```
Comment (krishnaraoch) [inline learn/wme-backup-restore/backup/platform/backup-aws.md:9]: Backup and Restore come under one file.
with two different types as I explained in the ESxi sheet

File: learn/wme-backup-restore/wme-backup-restore-introduction.md (hunk 2 above)
Comment (krishnaraoch) [reply learn/wme-backup-restore/wme-backup-restore-introduction.md:10]: Prequisities for Backup and Restore.
What happens if we take backup of Platform and StudioWorkspace instance in different times etc
Explain about two ways, Directories and Disk/Volumes

**Comment by prasanth-reddy-wavemaker-com** [comment]
whats next on this?

**Comment by wmdevops-os** [comment]
Will update and merge this branch in this week.

**Comment by vikassikakolli-wave** [comment]
Will update and merge this branch in this week.

**Pull request closed without merging**